Rev sd (#346)
cfrg/draft-irtf-cfrg-hash-to-curve: draft-irtf-cfrg-hash-to-curve.md

> # Security considerations {#security-considerations}

Reviewer: ```suggestion
# Security considerations {#security-considerations}

This section contains additional security considerations about the hash-to-curve mechanisms
described in this document.
```

a point on the curve, the expected number of tries is just two.
However, the running time of this method depends on the input string,

> which means that it is not safe to use in protocols sensitive to timing

Reviewer: ```suggestion
As such, it is not safe to use in protocols sensitive to timing
```
